decidim-api/app/controllers/decidim/api/graphiql_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Api
+    # Base controller for `decidim-api`. All other controllers inherit from this.
+    class GraphiQLController < ::GraphiQL::Rails::EditorsController
```

Comment:
Who's finally inheriting from this controller?

Reply:
I think I don't understand the question... No one's inheriting from `GraphiQLController`, we're using this controller to serve GraphiQL (check the routes file).

We're using this approach so we can hook into it and force the authentication of users in case the organization requires it, otherwise the access is closed to users but data is still accessible via the API.

Reply:
@mrcasals I think the comment is wrong, this is the controller for the graphiql IDE, not the api right?

Reply:
Ah yes, I hadn't seen the comment. It is indeed wrong, I'll change it, thanks!

Reply:
Done, check the last commit! Thanks!